fslock/flock_linux.go

```diff
@@ -0,0 +1,54 @@
+// Copyright 2022 by Dan Jacques. All rights reserved.
```

Comment:
nit: feel free to add yourself as a copyright holder here, up to you

Reply:
Done. Thanks : )